rasa_sdk/tracing/instrumentation/instrumentation.py

```diff
@@ -214,3 +242,58 @@ def mark_class_as_instrumented(instrumented_class: Type) -> None:
             _mangled_instrumented_boolean_attribute_name(instrumented_class),
             True,
         )
+
+
+def _instrument_validation_action_extract_validation_events(
+    tracer: Tracer,
+    validation_action_class: Union[Type[ValidationAction], Type[FormValidationAction]],
+) -> None:
+    def tracing_validation_action_extract_validation_events_wrapper(
+        fn: Callable,
+    ) -> Callable:
+        @functools.wraps(fn)
+        async def wrapper(
+            self,
+            dispatcher: "CollectingDispatcher",
+            tracker: "Tracker",
+            domain: "DomainDict",
```

Comment:
Because of the usage of `""` in the type hints here, you can import these at the top of the module using:
```
if TYPE_CHECKING:
    import ...
```

Reply:
I'm not even sure why it was used written that way, I just copied it from the method definition. I'll take it out.